producer-kafka/pom.xml

```diff
@@ -0,0 +1,257 @@
+          </dependency>
+        </dependencies>
+      </plugin>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>2.22.2</version>
+        <dependencies>
+          <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>${junit.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>unpack-sjk</id>
+            <phase>compile</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.gridkit.jvmtool</groupId>
+                  <artifactId>sjk-core</artifactId>
+                  <version>0.14</version>
+                  <outputDirectory>${project.build.directory}/sjk-core</outputDirectory>
+                </artifactItem>
+              </artifactItems>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-antrun-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>fix-sjk-core</id>
+            <phase>process-classes</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <tasks>
+                <!-- Remove root package classes -->
+                <delete>
+                  <fileset dir="${project.build.directory}/sjk-core/" includes="*.class"/>
+                </delete>
+                <jar destfile="${project.build.directory}/classes/sjk-core-0.14.jar">
+                  <fileset dir="${project.build.directory}/sjk-core"/>
+                </jar>
+                <delete dir="${project.build.directory}/sjk-core"/>
+              </tasks>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <unpackBundle>true</unpackBundle>
+          <instructions>
+            <Bundle-Name>Kafka-Producer</Bundle-Name>
+            <Bundle-Description>Kafka producer component for project</Bundle-Description>
+            <Bundle-SymbolicName>io.stargate.producer.kafka</Bundle-SymbolicName>
+            <Bundle-Activator>io.stargate.producer.kafka.KafkaProducerActivator</Bundle-Activator>
+            <Import-Package><![CDATA[
+              org.osgi.framework,
+              io.stargate.db,
+              ]]>
+            </Import-Package>
+            <Export-Package>io.stargate.producer.kafka</Export-Package>
```

Comment:
Do we expect any other bundle to link to the Kafka producer in OSGi? If not, I guess we do not have to export anything. WDYT?

Reply:
Yeah something top level like this shouldn't export anything.

Reply:
agree, this is a client application - I don't think that anyone will want to use it. I removed the `export-package`